.circleci/config.yml

```diff
@@ -117,7 +117,7 @@ defaults:
 
   - test_ubuntu1904: &test_ubuntu1904
       docker:
-        - image: ethereum/solidity-buildpack-deps:ubuntu1904
+        - image: ethereum/solidity-buildpack-deps:ubuntu1904_evmc7
```

Comment:
These images use evmone 0.2.0 so that's not evmc7 compatible.

Reply:
@bshastry what version of evmone is it using?

Reply:
I actually don't know the contents of `ubuntu1904_evmc7` so I cannot say which version of evmone it is using. Do you know how the image was created? Also, the naming scheme adopted by #7700 is `ethereum/solidity-buildpack-deps:ubuntu1904-1` where the revision is `1`, so it makes sense to change this to `ethereum/solidity-buildpack-deps:ubuntu1904-2`

Regarding evmc7, didn't we copy it into the solidity source repo and build it in tree (see https://github.com/ethereum/solidity/pull/7069#issuecomment-510361620)

Reply:
Please note that this PR also requires changes to the following docker files that are used to build images that are pushed to dockerhub and picked up by circle ci for builds/tests.

- `.circleci/docker/Dockerfile.ubuntu18.04`
- `.circleci/docker/Dockerfile.ubuntu19.04`
- `.circleci/docker/Dockerfile.clang.ubuntu19.04`

Reply:
To sum up, what this PR needs are the following things (probably among other things that I may be missing)
  - In all the docker files, bump up evmone version to `v0.3.0` that supports istanbul and is now released
  - Change the revision of all the docker images in the circleci config from `1` to `2`
  - Copy evmc7 source code to solidity repo, adapting it as required
  - Build, tag and push all docker images to docker hub by following the readme in the `.circleci` directory